data/test-data/prodConfigTest.yml

```diff
@@ -0,0 +1,47 @@
+label: 'test-production'
+
+default:
+  site: North
+  layoutName: Prod5
+  dataDirectory: /home/nb315/Work/Projects/CTA_MC/data-test
+  array:
+    modelVersion: Prod5
+    zenith: 20 deg
+    phi: 0 deg
+    default:
+      LST: 'D234'
+      MST: FlashCam-D
```

Comment:
It might make sense to have for North example files always NectarCam as default camera

Reply:
Yes, but I'd rather update this later. At the moment, I have to replace some files for the NectarCam to run. So I'd keep FlashCam as default for simplicity during this test phase, and then update it in the future.